DHS_Recode_V.do

```diff
@@ -188,7 +188,7 @@ capture confirm file "${INTER}/zsc_hm.dta"
 	
 gen c_placeholder = 1
 keep hv001 hv002 hvidx  ///
-a_* hm_* ln c_*
+a_* hm_* ln c_* hc70 hc71

```

Comment:
Do not write hc70 and hc71 here because the keep will not run if there's zsc file with birth.dta instead of running under the hm.dta.

Reply:
It cannot run through for countries without zsc file, if I don't add hc70 hc71. An error message will come out as hc70 not found. Do you know what may be the reason? You can try India2005 as an example

Reply:
I am faced with similar problems in IV. I suggest using "keep *hc70 *hc71", in this way it will just keep hm_hc70 hm_hc70 when running point with zsc file.

Reply:
Yeah, I agree with you!

Reply:
I am suggesting:
Do not write *hc70 *hc71 or anything contain hc70 or hc71. 

Because there are cases, where hc70 and hc71 do not produce from this domain.

Reply:
Thank you for pointing this out for us. I will rethink this. Would you mind giving an example of that situation so I could take a closer look on it? Thank you

Reply:
Thank you for pointing this out for us. I will rethink this. Would you mind giving an example of that situation so I could take a closer look at it? Thank you